src/frontend/src/lib/actors/actors.ic.ts

```diff
@@ -42,6 +43,36 @@ export const getBackendActor = async ({
 	return backend;
 };
```

Comment:
Here we create a new Signer actor.  The code is identical to the backend actor code except that it refers to the signer canister ID instead.  Dependencies such as internet identity can be stripped out from this copy soon.

Reply:
eheh! here we fall in the same topic: should we have a single function that takes as input the respective differentiations between `backend` and `signer`, such as canisterId, idlFactories, etc, to avoid repetition of code?

Reply:
I believe that the similarity in code is again likely to be short-lived.  For example, the internet identity check on line 54 will disappear for the signer.  That said, if you have a concrete suggestion for how to do this, I would be interested to learn.  In Rust I know how to pass a type such as `{Backend,Signer}Actor`, but not in typescript.

Reply:
indeed! it was just an observation; since it is all short-term, I would leave it like that and we adjust after, as you said